app/dao/LocationDao.scala

```diff
@@ -7,6 +7,7 @@ import dao.repo.LocationRepo
 import javax.inject.{Inject, Singleton}
 
 import scala.concurrent.{ExecutionContext, Future}
+import implicits.DateTimeTransform._
```

Comment:
Is it needed?

Reply:
yeah, it was in LocationTransforms, moved out of there